instrument/larmor/sans.py

```diff
@@ -372,63 +417,61 @@ def _configure_trans_custom(self):
 
     def _detector_is_on(self):
         """Is the detector currently on?"""
-        voltage_status = all([
+        voltage_status = [
             self.get_pv(
-                "CAEN:hv0:0:{}:status".format(x)).lower() == "on"
-            for x in [8, 9, 10, 11]])
-        return voltage_status
+                f"CAEN:hv0:0:{x}:status").lower() == "on"
+            for x in range(8, 12)]
+        return all(voltage_status)
 
     def _detector_turn_on(self, delay=True):
-        self.send_pv("CAEN:hv0:0:8:pwonoff", "On")
-        self.send_pv("CAEN:hv0:0:9:pwonoff", "On")
-        self.send_pv("CAEN:hv0:0:10:pwonoff", "On")
-        self.send_pv("CAEN:hv0:0:11:pwonoff", "On")
+        for i in range(8, 12):
+            self.send_pv(f"CAEN:hv0:0:{i}:pwonoff", "On")
+
         if delay:
             info("Waiting For Detector To Power Up (180s)")
             sleep(180)
 
     def _detector_turn_off(self, delay=True):
-        self.send_pv("CAEN:hv0:0:8:pwonoff", "Off")
-        self.send_pv("CAEN:hv0:0:9:pwonoff", "Off")
-        self.send_pv("CAEN:hv0:0:10:pwonoff", "Off")
-        self.send_pv("CAEN:hv0:0:11:pwonoff", "Off")
+        for i in range(8, 12):
+            self.send_pv(f"CAEN:hv0:0:{i}:pwonoff", "Off")
+
         if delay:
             info("Waiting For Detector To Power Down (60s)")
             sleep(60)
 
     # Instrument Specific Scripts
     @staticmethod
-    def FOMin():  # pylint: disable=invalid-name
+    def frame_overload_mirror_in():  # pylint: disable=invalid-name
         """Put the frame overload mirror into the beam."""
         # gen.cset(pol_trans=0, pol_arc=-1.6)
         # Convert to angle instead of mm
         gen.cset(pol_trans=0, pol_arc=-0.084)
 
     @staticmethod
-    def ShortPolariserin():  # pylint: disable=invalid-name
+    def short_polariser_in():  # pylint: disable=invalid-name
```

Comment:
Again, name changes like this will change the interface for the users at the beam line.  As long as you have the Larmor instrument scientists agreeing to the change, I'm happy with it.